qframe_test.go

```diff
@@ -1373,6 +1373,56 @@ mon
 	})
 }
 
+func TestQFrame_ReadCSVMissingColumnName(t *testing.T) {
+	input := `,COL2
+a,1.5
+b,2.25
+c,3.0`
+	expectedIn := `col,COL2
+a,1.5
+b,2.25
+c,3.0`
+
+	out := qframe.ReadCSV(strings.NewReader(input), csv.MissingColumnNameAlias("col"))
+	expected := qframe.ReadCSV(strings.NewReader(expectedIn))
+	assertEquals(t, expected, out)
+
+	// Output:
```

Comment:
Remove this comment or migrate this test to be an example test. See the examples in "examples_test.go". Those are part of the documentation and adding the happy/base cases there may be a good idea.